src/main/java/com/three_tech_solutions/slot_app/data/mappers/PriceMapper.java

```diff
@@ -1,11 +1,11 @@
 package com.three_tech_solutions.slot_app.data.mappers;
 
-import com.three_tech_solutions.slot_app.controllers.responses.PriceUpdateResponse;
+import com.three_tech_solutions.slot_app.controllers.responses.PriceResponse;
 import com.three_tech_solutions.slot_app.data.models.Price;
 
 public class PriceMapper {
-    public static PriceUpdateResponse toPriceUpdateResponse(Price price) {
-        return new PriceUpdateResponse(price.getId(), price.getName(), price.getAmount());
+    public static PriceResponse toPriceUpdateResponse(Price price) {
```

Comment:
En vez de crear un método estático es mejor si a la clase la creamos como un componente de Spring y dejamos que el framework maneje la inyección de dependencias


```suggestion
@Component
public class PriceMapper {
    public PriceResponse toPriceUpdateResponse(Price price) {
```